config/build.yml

```diff
@@ -1,3 +1,8 @@
+acsf:
```

Comment:
Instead of creating a new top-level key for `acsf` sites, I think this should use the existing `multisites` array below. This will allow you to run commands like `blt sync:db:all` for ACSF sites as well (at some point in the future when multiple local ACSF sites are supported, anyway)